students/5/readme.md

```diff
@@ -9,3 +9,5 @@ My favorite food is Eru.
 My favorite band is Moses Bliss...
 
 My favorite holiday is christmas.
+
+My favorite season is winter
```

Comment:
Please put a period after the word winter and capitalize Winter.

Reply:
Okay Sir.